_announcements/2020-03-25-announcement-of-emergency-board-meeting.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Announcement of Emergency Board Meeting
+date: 2020-03-25 20:45
+author: Klahanie
+comments: true
+categories: [announcements]
+show-title: true
+---
+
+Due in part to the Coronavirus and the Governor's announcement, the Board is having an Emergency Meeting this Saturday at 1:00pm. The meeting will be totally remote, but open to residents. The Zoom conferencing software will be used to provide access to the open public portion of the meeting. This is designed for video as well as phone in connections. The session is currently allocated for the 1:00pm. start and close before 2:30pm.

```

Comment:
We are quite a ways from a Klahanie style guide, but it doesn't hurt to start agreeing on some consistencies, like how we want to abbreviate ante/post meridiem. I mostly use the Chicago Manual of Style (CMoS) and they recommend "p.m." This page uses two different abbreviations. "...this Saturday at 1:00 p.m."